tests/test_prepdocs.py

```diff
@@ -84,12 +107,23 @@ async def mock_create(*args, **kwargs):
     ]
 
 
+def fake_response(http_code):
+    return Response(http_code, request=Request(method="get", url="https://foo.bar/"))
```

Comment:
The foo.bar here also seems like placeholder, maybe make this look like an openai URL?